src/miner.cpp

```diff
@@ -71,10 +71,10 @@ bool TxPriorityCompare::operator()(const TxPriority& a, const TxPriority& b)
             const CScCertificate& aCert = dynamic_cast<const CScCertificate&>(*a.get<2>());
             const CScCertificate& bCert = dynamic_cast<const CScCertificate&>(*b.get<2>());
 
-            if (aCert.GetScId() == bCert.GetScId() )
-            {
-                return aCert.quality > bCert.quality;
+            if (a.get<1>() == b.get<1>()) { 
+                return aCert.quality < bCert.quality;
```

Comment:
Can be a third-level sorting based on priority worth? I am referring to the case where both a fee tie and a quality tie occur.